styleguide/_themes/global/scss/global.scss

```diff
@@ -5,3 +5,4 @@
 @import '_fonts';
 @import '_forms';
 @import '_buttons';
+@import '_bootstrap_overrides';
```

Comment:
Another PR that is just a bunch of filename changes (no line changes) is my preference. This is good to go.